registry/session.go

```diff
@@ -281,6 +281,9 @@ func (r *Session) GetRepositoryData(remote string) (*RepositoryData, error) {
 	// TODO: Right now we're ignoring checksums in the response body.
 	// In the future, we need to use them to check image validity.
 	if res.StatusCode != 200 {
+		if res.StatusCode == 404 {
```

Comment:
I will update the nesting. The use of constants over numbers should be done in a refactor PR, I will keep this change consistent with the rest of the code.